packages/grafana-o11y-ds-frontend/src/TemporaryAlert.tsx

```diff
@@ -30,22 +30,26 @@ const timeoutMap = {
 export const TemporaryAlert = (props: { severity: AlertVariant; text: string }) => {
```

Comment:
I would love to have a unit test of this component. Doesn't have to be in this PR, but it would be great.

Reply:
Indeed! That's also what I meant with:
> Of course this proves that we need tests for this component. I'll consider adding them if I find some spare time.

Reply:
😅 sorry, I was reviewing during a meeting.

Reply:
No worries! ❤️